ArduPlane/defines.h

```diff
@@ -145,6 +145,10 @@ enum {
     USE_REVERSE_THRUST_FBWB                     = (1<<9),
     USE_REVERSE_THRUST_GUIDED                   = (1<<10),
     USE_REVERSE_THRUST_AUTO_LANDING_PATTERN     = (1<<11),
+    USE_REVERSE_THRUST_FBWA                   = (1<<12),
+    USE_REVERSE_THRUST_ACRO                   = (1<<13),
+    USE_REVERSE_THRUST_STABILIZE                   = (1<<14),
+    USE_REVERSE_THRUST_THERMAL            =(1<<15),
```

Comment:
need to also widen use_reverse_thrust to 32 bits, as its signed. We have an easy function now for converting widened parameters

Reply:
@tridge || @peterbarker  will need a pointer to a pattern for what you want me to do to change it to 32 wide... thanks

Reply:
fixed..